store/sqlstore/upgrade.go

```diff
@@ -1297,3 +1298,13 @@ func fixCRTChannelMembershipCounts(sqlStore *SqlStore) {
 		mlog.Error("Error marking migration as done", mlog.Err(err))
 	}
 }
+
+func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
+	// if shouldPerformUpgrade(sqlStore, Version5380, Version600) {
+
+	// allow 10 files per post
+	sqlStore.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
```

Comment:
Do you think it's worth to diverge the schema to use native array types for Postgres? We can query the data more efficiently in those cases.

The querying side might have to change, so there will be a cost I think.

Reply:
Technically we could have converted to JSON this one as well since that's what we store in it but I opted not to since we have plans to remove the column ([MM-32754](https://mattermost.atlassian.net/browse/MM-32754)), although I am not 100% convinced yet and currently thinking of deferring it to a later time.